site/docs/09-math/07-vector.mdx

```diff
@@ -61,3 +61,169 @@ console.log(anotherPoint.toString()) // "(50, 50)"
 ```
 
 Notice how both `point` and `samePoint` share the same vector reference, so using `setTo` mutates the vector. Use `clone` to ensure you are not changing vectors unexpectedly!
+
+### Static Methods Available
```

Comment:
Including the APIs here in the docs is nice for seeing it inline, but I'm also concerned this duplicates effort/information with the Typedoc API docs. If the signatures change, people need to remember to update them here which is what we want to avoid.

1. Is the same information (with examples) in the JSDoc for each of the methods listed here? If not, I think it should be.
2. Could we instead maybe link to each of them versus including them all here?

It makes me think that it would be handy to have a way to "include" API docs in MDX files here.

Reply:
i understand the why behind this... will update to links